app/frontend/components/AmountSelection/AmountSelection.js

```diff
@@ -4,6 +4,7 @@ import React, { Component } from 'react';
 import { FormattedMessage } from 'react-intl';
 import DonationBands from '../DonationBands/DonationBands';
 import Button from '../Button/Button';
+import $ from '../../util/PubSub';
```

Comment:
We can actually use `import $ from 'jquery'` since it's alised to `window.jQuery` (so it will use champaign's `$` global with `.publish` already available).

In other words, is `PubSub.js` duplicating our efforsts?